packages/patternfly-4/react-core/src/components/Switch/Switch.docs.js

```diff
@@ -0,0 +1,13 @@
+import { Switch } from '@patternfly/react-core';
+import SimpleSwitch from './examples/SimpleSwitch';
+import NoLabelSwitch from './examples/NoLabelSwitch';
+import DisabledSwitch from './examples/DisabledSwitch';
+import UncontrolledSwitch from './examples/UncontrolledSwitch';
+
+export default {
+  title: 'Switch',
+  components: {
+    Switch
+  },
+  examples: [SimpleSwitch, NoLabelSwitch, DisabledSwitch, UncontrolledSwitch]
```

Comment:
This was changed recently, static fields should be moved out of the examples and should now be
```suggestion
  examples: [{ component: SimpleSwitch, title: 'Simple Switch' }, { component: NoLabelSwitch, title: 'Switch with no labels' }, { component: DisabledSwitch, title: 'Disabled Switch' }, { component: UncontrolledSwitch, title: 'Uncontrolled Switch' }]
```

Reply:
Thanks